erpnext/selling/report/customer_sales_target_not_met/customer_sales_target_not_met.py

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2013, Frappe Technologies Pvt. Ltd. and contributors
+# For license information, please see license.txt
+
+from __future__ import unicode_literals
+from frappe import _
+import frappe
+
+def execute(filters=None):
+	columns=get_columns(filters)
+
+	data = get_data(filters)
+
+	return columns, data
+
+def get_columns(filters):
+	return [
+        _("Customer") + ":Link/Customer:150",
+		_("Monthly Sales Target") + ":currency:150",
+		_("Total Monthly Salest") + ":currency:150"
+        ]
```

Comment:
- Fix the indentation (use only tabs)
- Use dict definition for the columns, instead of simple strings